### PR TITLE
feat(applications): handle `Path` field in desktop entries

### PR DESCRIPTION
**Pull request by loqusion**
Some desktop entries are required to be run under a certain directory; for instance, the desktop entries created by https://github.com/quickemu-project/quickemu.

I'm new to Rust, so any feedback is appreciated.

### CODE REVIEW

File: plugins/applications/src/scrubber.rs [hunk 1]
```diff
@@ -73,6 +74,7 @@ impl DesktopEntry {
                                 }
                                 exec
                             },
+                            path: map.get("Path").map(|val| PathBuf::from(val)),
```
Comment (Kirottu) [inline plugins/applications/src/scrubber.rs:77]: Can be simplified to be `map.get("Path").map(PathBuf::from)`.

File: plugins/applications/src/lib.rs [hunk 2]
```diff
@@ -65,7 +65,12 @@ pub fn handler(selection: Match, state: &State) -> HandleResult {
                 }
             }
         }
-    } else if let Err(why) = Command::new("sh").arg("-c").arg(&entry.exec).spawn() {
+    } else if let Err(why) = Command::new("sh")
+        .arg("-c")
+        .arg(&entry.exec)
+        .current_dir(entry.path.as_ref().unwrap_or(&env::current_dir().unwrap()))
```
Comment (Kirottu) [inline plugins/applications/src/lib.rs:71]: Indentation seems a bit wonky here, format the code with `cargo fmt`.

**Review by Kirottu — COMMENTED** [review]
Looks generally fine except for a few code style issues.

File: plugins/applications/src/lib.rs (hunk 2 above)
Comment (loqusion) [reply plugins/applications/src/lib.rs:71]: I ran `cargo fmt` and it's still the same

**Comment by Kirottu** [comment]
Alright, looks good to me!

**Pull request merged**